Tests/Serializer/JMSSerializerAdapterTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Tests\Serializer;
+
+use FOS\RestBundle\Context\Context;
+use FOS\RestBundle\Serializer\JMSSerializerAdapter;
+use JMS\Serializer\ContextFactory\DeserializationContextFactoryInterface;
+use JMS\Serializer\ContextFactory\SerializationContextFactoryInterface;
+use JMS\Serializer\DeserializationContext;
+use JMS\Serializer\Exclusion\ExclusionStrategyInterface;
+use JMS\Serializer\SerializationContext;
+use JMS\Serializer\SerializerInterface;
+use PhpCollection\MapInterface;
+
+class JMSSerializerAdapterTest extends \PHPUnit_Framework_TestCase
+{
+    private $serializer;
+    private $serializationContextFactory;
+    private $deserializationContextFactory;
+    private $adapter;
+
+    protected function setUp()
+    {
+        $this->serializer = $this->getMockBuilder(SerializerInterface::class)->getMock();
+
+        $this->serializationContextFactory = $this
+            ->getMockBuilder(SerializationContextFactoryInterface::class)->getMock();
+        $this->deserializationContextFactory = $this
+            ->getMockBuilder(DeserializationContextFactoryInterface::class)->getMock();
+
+        $this->adapter = new JMSSerializerAdapter(
+            $this->serializer,
+            $this->serializationContextFactory,
+            $this->deserializationContextFactory
+        );
+    }
+
+    public function testBasicSerializeAdapterWithoutContextFactories()
+    {
+        $jmsContext = SerializationContext::create();
+        $adapter = new JMSSerializerAdapter($this->serializer);
+        $this->serializer->expects($this->once())->method('serialize')->with('foo', 'json', $jmsContext);
+
+        $adapter->serialize('foo', 'json', new Context());
+    }
+
+    public function testBasicDeSerializeAdapterWithoutContextFactories()
+    {
+        $jmsContext = DeserializationContext::create();
+        $adapter = new JMSSerializerAdapter($this->serializer);
+        $this->serializer->expects($this->once())->method('deserialize')->with('foo', 'json', $jmsContext);
```

Comment:
ah, it did not https://travis-ci.org/FriendsOfSymfony/FOSRestBundle/jobs/235667634 :confused:  :confused:  :confused:  :confused: